### DISCUSSION TITLE
Balancing : Daily reputation penalty for dominated planets governments

### DISCUSSION THREAD

**Discussion by Kheldjin** [endless-sky/endless-sky#10255] [Ideas]
Balancing suggestion for long term domination mechanic use.

Add a small amount of reputation loss for each tribute credit received from planetary government because they wouldn't usualy forgive you for extorting them just because you slaughtered pirates on the other side of the galaxy.

Possible issues with government changes could be handled by having a minimum tribute threshold to trigger (10 cred?) and lowering tribute values as done during FW campain: 
- during occupation (FW campain)
- until expiration of a landing mission warning you of a definitive government change on a planet (check if tribute when government change) in order to let you have the time to come relinquish tribute.
- Alternatively to the previous : direct remote relinquishing query on landing with no changes to tribute value

**Comment by TheGiraffe3** [comment]
@TomGoodIdea would this be possible to put into practice?

**Reply by TomGoodIdea** [reply]
Yeah, sure. Tributing already results in a one-time penalty, but daily reputation loss would possibly discourage rep farming.

**Reply by TheGiraffe3** [reply]
I mean code wise could it be done? ~also it would incentivize me not to finish FW so I don’t lose rep with the Republic~

**Comment by mOctave** [comment]
I'm not sure how I would feel about a constant rep penalty, since then there's nothing stopping you just dominating every planet you see and cutting your losses, but it would also pretty much break the game to go fight the Pug with a hostile Republic. Maybe if there was a rep penalty of something like `max(0.1, 0.1sqrt(n))` points per day per planet, where n is your current rep? Then it's not such a big deal when you're hostile already, but the Republic's never going to trust you much even after FW if you have all their planets.

**Reply by Kheldjin** [reply]
I hadn't thought about the campain friendly escorts but there is an easy fix for that.
You would only need to use the unused "Republic (Friendly)" government and equivalents for FW and Syndicate on the escort and invasion fleets. Maybe put a fail condition if your reputation drops with the "Friendly" governments to avoid them being free capture targets.

RP wise it could be considered them reluctantly joining forces with you exclusively to counter the pug and only if you don't mess with the forces they send with you.

Otherwise you could use a reputation check to create a branch where they send you alone in hostile territory because they can't trust you (maybe offer to lease you an inferior jumpdrive equipped Syndicate mercenary group for a hefty sum), and consider it an added difficulty.
In this case, I would still switch the invasion fleets as "Friendly" governments so they don't shoot you when they arrive.

If you are worried about breaking the mission chain, I would also recommand adjusting the "Navy intelligence" government to ensure you can't be directly hostile with the only ship using it which is supposed to escort you as well (reputation reset, ship switch to "Republic (Friendly)", or government attitude changes).

**Comment by ghost** [comment]
If that would get into the game, I think a reminder (when the planet changes factions) would be a good idea.
For example, in my earlier save, I dominated all the pirate worlds before starting the Free Worlds campaign.
That is all fine, they are pirates and hostile anyway, who cares, if they hate me.
But during the FW campaign, some of the pirate worlds either joined the FW or became neutral.
Since they weren't pirates anymore, I stopped demanding the tribute from them for RP reasons etc.
But, if someone didn't, and didn't know the reputation loss is a thing (if they only tributed pirates like I did, they could very well never find out), they could then be taken by surprise, when the Free Worlds suddenly would become hostile for no reason that they knew of.
But a pop-up ("X planet changed allegiance, tributing them would cause reputation loss with their new faction. Do you want to stop demanding it?" and Accept/Decline buttons) should help with that.

**Reply by mOctave** [reply]
Even just something written into the missions, maybe, like a Republic captain who refuses to work with you if you've tributed the Republic.

**Reply by Saugia** [reply]
Keep in mind that the tribute mechanic is entirely non-canon and meant as a mini-game after the fact. There is now a warning for every time you attempt to dominate a planet to let you know that it can harm your rep with a faction and cause issues; essentially saying you acknowledge it could hurt campaign missions or the likes.

We wouldn't intentionally integrate planetary domination/tribute with vanilla content; it's been brought up before and we've said we want to keep it separate, in part due to the purpose being outside of the story, and because it would be incredibly difficult to track that sort of thing for every single mission or story content and make adjustments based upon it. It could theoreticality be done, but we don't wish to do so within vanilla; that said, it would be good for plug-in use.

**Reply by TheGiraffe3** [reply]
Though there is the fact that some people typically dominate all Republic planets during FW, so I'm not sure it's particularly non-canon. 

But good point.

**Reply by Saugia** [reply]
It is 100% not canon. The option is there, and you can do it it at any time you have the combat rating, but it intentionally is not within the canon part of the game's story, regardless of what the player does on their end.

The mechanic is a callback/implementation of the same mechanic in EV; in EV, the mechanic could also easily break story, and was meant more as something to do after you've completed the story as there was nothing else to do (besides jobs). You do it to dominate the galaxy and gain a bunch of credits, flexing what you have dominated, challenging fights, obtain ships from the fights, etc.. 

(Though, the notable difference being in EV there was a hard fleet limit of 6 ships. Of course it won't be difficult if you're flying around with a massive fleet, but ES's intended fleet gameplay is loosely based around that number as well.)

**Reply by ghost** [reply]
Fair about the non-canon part, but since it is part of the gameplay, and the player might be unaware of the constant rep loss while dominating someone (if that will be added to the game) if they only dominated the already hostile factions (like pirates), I think the warning about it, if the dominated planet changes allegiance, would still be a good thing.

**Comment by DoublePants** [comment]
I'd add a condition that if you've dominated a certain number of a faction's worlds, you cease losing that reputation and reset rep with them, as you become effectively their faction's overlord.
